### PR TITLE
Contact section

### PR DESCRIPTION
**Pull request by agustinaciraco**
## Summary

Added contact section with input and submit button.

### Full screen
contact-section-full

### Small screen
contact-section-small

## Trello Card

[contact-section] https://trello.com/c/OfkRskkx/3-maquetar-el-footer-de-la-pagina.

### CODE REVIEW

**Comment by SKOLZ** [comment]
rebase this branch please

**Comment by agustinaciraco** [comment]
done the rebase.

File: maquetado/src/index.pug [hunk 1]
```diff
@@ -40,6 +40,12 @@ html(lang='en')
           | No internet needed
         p.feature-description
           | All your debts are kept safe in your phone.
+    .section.contact-section
+        h1.contact-title
```
Comment (SKOLZ) [inline maquetado/src/index.pug]: use 2 space indentation

File: maquetado/src/index.pug [hunk 2]
```diff
@@ -40,6 +40,12 @@ html(lang='en')
...
+          | Get info on new updates
+        form.contact-form(name='contact-form')
+          input#email.form-control(type='email')
+          button.btn.btn-primary(type='submit') Subscribe
```
Comment (SKOLZ) [inline maquetado/src/index.pug]: separate content from html tags:
```
button.btn.btn-primary(type='submit')
  | Subscribe

File: maquetado/src/scss/base.scss [hunk 3]
```diff
@@ -63,6 +63,48 @@ body {
   position: absolute;
   top: -12px;
   width: 100%;
+  }
+
```
Comment (SKOLZ) [inline maquetado/src/scss/base.scss]: extra empty line

File: maquetado/src/scss/base.scss [hunk 4]
```diff
@@ -63,6 +63,48 @@ body {
...
+    font-size: 25px;
+  }
+
+  .contact-form {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+  }
+
+  input[type=email] {
```
Comment (SKOLZ) [inline maquetado/src/scss/base.scss]: add classes and style the classes, not the elements.

File: maquetado/src/scss/base.scss [hunk 5]
```diff
@@ -63,6 +63,48 @@ body {
...
+    border-radius: 10px;
+    margin: 10px;
+    padding-left: 60px;
```
Comment (SKOLZ) [inline maquetado/src/scss/base.scss]: I would write all the padding properties in a single one like this:
```
padding: 25px 60px 0;
```
when using 3 params the second one represents left and right

File: maquetado/src/scss/base.scss [hunk 6]
```diff
@@ -63,6 +63,48 @@ body {
...
+    padding-right: 60px;
+    padding-top: 25px;
+  }
+
+  button[type=submit] {
```
Comment (SKOLZ) [inline maquetado/src/scss/base.scss]: add classes and style the classes, not the elements.

File: maquetado/src/scss/base.scss [hunk 7]
```diff
@@ -63,6 +63,48 @@ body {
...
+    background-color: $orange;
+    border: 2px solid $white;
+    border-radius: 10px;
+    color: $white;
+    font-size: 110%;
+    font-weight: bold;
+    margin: 10px;
+    padding-bottom: 10px;
```
Comment (SKOLZ) [inline maquetado/src/scss/base.scss]:
```
padding: 10px 22px;
```

1st param: top and bottom.
2nd param: left and right.

File: maquetado/src/scss/base.scss [hunk 8]
```diff
@@ -65,6 +65,42 @@ body {
   width: 100%;
 }
 
+.contact-section {
+  background-color: $orange;
+  border-image: linear-gradient($gray-bg, $grad-dark, $gray-bg) 50 round;
```
Comment (SKOLZ) [inline maquetado/src/scss/base.scss]: you can use box-shadow to achieve this :P

File: maquetado/src/index.pug [hunk 9]
```diff
@@ -40,6 +40,13 @@ html(lang='en')
           | No internet needed
         p.feature-description
           | All your debts are kept safe in your phone.
+    .section.contact-section
+      h1.contact-title
+        | Get info on new updates
+      form.contact-form(name='contact-form')
+        input.input-field#email.form-control(type='email')
```
Comment (SKOLZ) [inline maquetado/src/index.pug]: set the id as the first attribute:
```
input#email.input-field.form-control(type='email')

File: maquetado/src/scss/base.scss [hunk 10]
```diff
@@ -65,6 +65,42 @@ body {
...
+    margin: 10px;
+    padding: 25px 60px 0;
+  }
+
+  .contact-button {
+    background-color: $orange;
+    border: 2px solid $white;
+    border-radius: 10px;
+    color: $white;
+    font-size: 110%;
```
Comment (SKOLZ) [inline maquetado/src/scss/base.scss]: use px font size

**Pull request merged**